projects/plugins/jetpack/_inc/client/traffic/related-posts.jsx

```diff
@@ -47,6 +47,44 @@ class RelatedPostsComponent extends React.Component {
 		analytics.tracks.recordJetpackClick( 'configure-related-posts' );
 	};
 
+	renderConfigureLink() {
+		const { isBlockThemeActive, lastPostUrl, siteAdminUrl } = this.props;
+
+		if ( isBlockThemeActive ) {
+			return (
+				<Card
+					compact
+					className="jp-settings-card__configure-link"
+					onClick={ this.trackConfigureClick }
+					href={ `${ siteAdminUrl }site-editor.php` }
```

Comment:
This PR tested well, though I do wonder if linking to the support doc would be more helpful overall? Just dropping someone into the site editor could be a bit overwhelming especially if they have not used it before.

https://jetpack.com/support/related-posts/#adding-related-posts-block-theme

Reply:
You're right, that may be a bit disconcerting for folks not used to the site editor yet. I'm happy to link to the support doc if that helps. I just pushed c51ea7cab49f84241c182ca8bde5fca4b86fa38e with that.